portfolio/src/main/java/com/google/sps/servlets/TestServlet.java

```diff
@@ -10,9 +12,19 @@
 @WebServlet("/introduction")
 public class TestServlet extends HttpServlet {
 
+
   @Override
-  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello! My name is Nolan</h1>");
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    //Array List 
+    ArrayList<String> listofmessage = new ArrayList<String>();
```

Comment:
This can be `List<String> messages = new ArrayList<String>();`
`ArrayList` is implementation detail, we only need to expose the most generic type.